draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx

```diff
@@ -257,7 +261,7 @@ TwoStackedLayout.storyName = "Two Stacked Content Layout"
 export const StackedLayoutWithScene = () => (
   <GuidanceBlock
     layout="stacked"
-    illustration={<HumanityAtWork alt="" />}
+    illustration={<SkillsCoachManagerHub alt="" />}
```

Comment:
Thanks for adding the portrait illustration 👍 should be able to sort that out in the aspect ratio fix